packages/generator-single-spa/src/common-templates/typescript/declarations.d.ts

```diff
@@ -2,3 +2,44 @@ declare module "*.html" {
   const rawHtmlFile: string;
   export = rawHtmlFile;
 }
+
+declare module "*.bmp" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.gif" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpeg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.png" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.webp" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.svg" {
+  import * as React from "react";
+
+  export const ReactComponent: React.FunctionComponent<
```

Comment:
Will this throw errors in utility modules that don't have react installed? we can change declarations.d.ts to be declarations.d.ts.ejs and then use `<% if (react) { $>` to include this in react projects but not in other projects.

Reply:
Check, I'm so used to React, that I didn't even spotted this one. Fixed as proposed.